keyboards/planck/planck.c

```diff
@@ -16,4 +19,15 @@ void matrix_init_kb(void) {
 	PORTE |= (1<<6);
 
 	matrix_init_user();
-}
\ No newline at end of file
+}
+
+#ifdef RAW_ENABLE
+
+void raw_hid_receive( uint8_t *data, uint8_t length )
+{
+	// Basic test of Raw HID
+	// Echo back data received
+	raw_hid_send( data, length );
+}
+
+#endif
```

Comment:
I think this could go in `quantum.c`, right? If it's still being tested, maybe just in a keymap?